pkg/handler/handler.go

```diff
@@ -335,6 +337,37 @@ func (h handler) apbAddSpec(w http.ResponseWriter, r *http.Request, params map[s
 	writeDefaultResponse(w, http.StatusOK, resp, err)
 }
 
+func (h handler) apbRemoveSpec(w http.ResponseWriter, r *http.Request, params map[string]string) {
+	ansibleBroker, ok := h.broker.(broker.DevBroker)
+	if !ok {
+		h.log.Errorf("unable to use broker - %T as ansible service broker", h.broker)
```

Comment:
I have opinions. The f variant is what is correct according to the docs IMO
https://github.com/op/go-logging/blob/master/logger.go#L253